google/cloud/google_cloud_cpp_common.cmake

```diff
@@ -171,7 +171,7 @@ target_link_libraries(
            Threads::Threads
            OpenSSL::Crypto)
 
-if (experimental-opentelemetry IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
+if (opentelemetry IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
```

Comment:
If you still support `experimental-opentelemetry`, then you cannot remove it from here either.

Reply:
See above. We replace `experimental-opentelemetry` with `opentelemetry`.